public/js/cita.json

```diff
@@ -1926,7 +1926,7 @@
   },
   {
     "title": "Maidstone",
-    "address": "2 Bower Terrace\nTonbridge Road\nMAIDSTONE\nKent\nME16 8RY",
+    "address": "2 Bower Terrace\nTonbridge Road\nMaidstone\nKent\nME16 8RY",
```

Comment:
I did this because the other Maidstone location is title case so I though it would look a bit weird when the two are listed together.